### ISSUE TITLE
Template: Search has focus when visibly hidden - 2.4.7 Focus Visible 

### ISSUE THREAD

**Issue by syarwick** [loyolauniversitychicago/loyola-template-4.0#90]
For the Search modal, both the"Search" and "Submit" are not visibly shown when they have focus, but are available to screen reader users (the search can be completed while it has this focus). 

Ensure focused items are visible AND items that are not visible do not receive focus.

**Comment by syarwick23** [comment]
Issue fixed with new template
